webapp-django/crashstats/crashstats/urls.py

```diff
@@ -104,7 +104,8 @@
     url(
         r'^status/$',
         RedirectView.as_view(
-            pattern_name='monitoring:index'
+            pattern_name='monitoring:index',
+            permanent=perm_legacy_redirect
         ),
         name='status_redirect',
     ),
```

Comment:
I'm not even sure we need this endpoint at all. It's probably for something that changed ages ago. We can probably just drop it altogether.